src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/Engine/Trivia/CSharpTriviaFormatter.cs

```diff
@@ -354,5 +354,20 @@ private SyntaxTrivia FormatDocumentComment(LineColumn lineColumn, SyntaxTrivia t
 
             return GetLineColumnDelta(lineColumn, docComment);
         }
+
+        protected override bool IsLineContinuation(SyntaxTrivia trivia)
+        {
+            return false;
+        }
+
+        /// <summary>
+        /// This is only here to make interface happy
```

Comment:
What does this mean? Does it mean the method is somehow unreachable?

Reply:
yeah, i don't understand this comment either.

Reply:
It is unreachable in the current implementation (is it only checked after finding a LineContinuation Character )but didn't know how I should handle it if in the future the C# formatter want to use it. Please suggest the proper way to handle.

Reply:
You can combine `IsLineContinuation` and `IsComment` to a single method `IsLineContinuationFollowedByComment`. That allows the C# implementation to return false with a simple comment that it does not have line continuations.

Reply:
@sharwell I still need IsComment lower down and while I like the idea of IsLineContinuationFollowedByComment I would need to pass the list and a start index or the previous and next trivia to it. If that is OK its an easy change and makes the code look cleaner. Alternatively **_I think but don't know how_** for the abstract class to implement both functions and return False and then the VB implementation would just stay the way it is. I would rename the function to IsVBComment or something like that. I could also just rename IsComment to IsVBComment and return false from C# code and avoid confusion. Waiting for preferred approach,

Reply:
Done